api/src/main/resources/workbench-api.yaml

```diff
@@ -7711,3 +7729,60 @@ definitions:
         description: Datetime at creation of this user's accont
         type: string
         format: date-time
+  ReportingResearcher:
+    description: >
+      Payload for the researcher table in the BigQuery reporting dataset.
+    type: object
+    properties:
+      username:
+        description: Username (GSuite)
```

Comment:
Please clarify whether this includes the domain

Reply:
yes it does, though now that I think about it, we'll be documenting the schema separately from this file so maybe I should pull these descriptions from here.

We should probably make a new yaml file for internal, backend-only codegen, but it's not worth it compared to just using Immutables.